From 49a76a526c8c4ad9245a999e425c1d9b7810d7df Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Thu, 8 Aug 2013 20:56:57 +0200
Subject: [PATCH 39/65] acerhk: fix build warnings on x86_64

the bios_routine and cmos_index variables are not used for x86_64.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 drivers/platform/x86/acerhk.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/platform/x86/acerhk.c
+++ b/drivers/platform/x86/acerhk.c
@@ -175,11 +175,13 @@ static void *reg1;
 /* mapped IO area from 0xe0000 */
 static void *reg2;
 
+#ifndef __x86_64__
 /* location of IO routine in mapped area */
 static unsigned int bios_routine;
 
 /* index of CMOS port to get key event */
 static unsigned int cmos_index;
+#endif
 
 /* function for bios call */
 static bios_call call_bios;
